js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -193,7 +193,7 @@ usersAdded_one={{count}} user added to the group
 resourcesAndScopes=Resources and Scopes
 editUsernameHelp=If enabled, the username field is editable, readonly otherwise.
 eventTypes.UPDATE_CONSENT_ERROR.description=Update consent error
-overrideActionTokensHelp=Override default settings of maximum time before an action permit sent by a user (such as a forgot password e-mail) is expired for specific action. This value is recommended to be short because it's expected that the user would react to self-created action quickly.
+overrideActionTokensHelp=Override default settings of maximum time before an action permit sent by a user (such as a forgot password e-mail) is expired for specific action. This value is recommended to be short because it is expected that the user would react to self-created action quickly.
```

Comment:
```suggestion
overrideActionTokensHelp=Override default settings of maximum time before an action permit sent by a user (such as a forgot password e-mail) is expired for a specific action. This value is recommended to be short because it is expected that the user would react to a self-created action quickly.
```